.prettierrc

```diff
@@ -1,6 +1,6 @@
 {
-  "trailingComma": "none",
+  "singleQuote": true,
```

Comment:
Why change this?

Reply:
Huh, I think this may have been some "me" technical difficulty, because I didn't have a local .prettierrc but it may have been something I inadvertently did. I was trying to add one so that my local linting would actually work.

Lemme go check out a clean version and figure out what is going on!

Reply:
Totally my fault, it should be reverted now!